backend/src/main/java/com/bakdata/conquery/models/datasets/concepts/ConceptElement.java

```diff
@@ -16,7 +16,6 @@
 import lombok.Setter;
 import lombok.ToString;
 
-@ToString(callSuper = true)
```

Comment:
Hast du die ToString dann in den children implementiert?

Reply:
Die IDE hat mir die Annotation als ignoriert angezeigt, weil es schon eine toString in der Klasse gibt. Die Children nutzen diese dann